cmd/gokeyless/gokeyless.go

```diff
@@ -216,7 +216,7 @@ func main() {
 		fmt.Println("gokeyless version", version)
 		os.Exit(0)
 	case manualMode && configMode:
-		log.Fatal("can't specify both -manual-activation and -config-only!")
+		log.Fatal("can't specify both --manual_activation and --config_only!")
```

Comment:
It actually accepts both forms ftr

Reply:
Oh, I fixed too much I guess. I was trying to solve `unknown shorthand flag: 'm' in -manual-activation`, but turns you need to do `--manual-activation` (or `--manual_activation`), `config-only` accidentally works with just one dash because it has `-c` registered as a shorthand. tl;dr I just need to change `-` to `--` in the error messages.

Reply:
👍